src/identity.js

```diff
@@ -1550,6 +1542,14 @@ export default function Identity(mpInstance) {
                     );
                 }
 
+                const incomingUser = self.IdentityAPI.getUser(
+                    identityApiResult.mpid
+                );
+
+                const incomingMpidUIByName = incomingUser
```

Comment:
i think `incomingMpidUIByName` can be shortened to `incomingUIByName`.  Elsewhere we have `previousUIByName`, so it's just being consistent with that.